gems/redis/4.2/_test/sorted_sets.rb

```diff
@@ -0,0 +1,11 @@
+redis = Redis.new
+
+redis.zcard(:sample_key)
+redis.zadd(:sample_key, 1, 'hello world')
+redis.zadd("string key", 1, 'hello world', nx: true)
+redis.zadd("string key", 2, 'hello world', xx: true)
+redis.zadd("string key", 3, 'hello world', ch: true)
+redis.zadd("string key", 4, 'hello world', incr: true)
+redis.zadd(:foo, [2, "s1", 3, "s2", 4, "s3"], xx: true)
+redis.zincrby(:foo, 1,  "bar")
```

Comment:
How about adding other cases, in addition to `Integer`?

```ruby
redis.zincrby(:foo, 1.0,  "bar")
redis.zincrby(:foo, "-inf",  "bar")
redis.zincrby(:foo, "+inf",  "bar")
```

Reply:
thanks!
add test case with https://github.com/ruby/gem_rbs_collection/pull/271/commits/eda2e166624ed88369a398b307ec8997cfe0b788